source/concepts/networking/network-firewall.html.md.erb

```diff
@@ -0,0 +1,31 @@
+---
+owner_slack: "#modernisation-platform"
+title: AWS Network Firewall
+last_reviewed_on: 2022-02-06
+review_in: 6 months
+---
+
+# <%= current_page.data.title %>
+
+We currently have setup in our core-network account AWS Network Firewall to monitor and control network traffic on a layer 3 level for transit gateway connection and cross account vpcs to protecte reources
```

Comment:
`protect resources` rather than `protecte reources`

Reply:
Still has a mis-spelling - protecte reources instead of protect resources